src/Components/Dice/AbilityIcon.tsx

```diff
@@ -0,0 +1,6 @@
+import { ReactElement } from "react";
+import Advantage from "../../svg/swrpg-advantage.svg";
+
+export const AbilityIcon = (props: { className: string }): ReactElement => {
```

Comment:
These limited icons do not allow for color theming. The scope of this PR demands a separate task to deal with this.